### PR TITLE
code-health: fix-typos

### PR DESCRIPTION
**Pull request by vr009**
With this patch:
- Added a new target with running codespell for checking typos in repo.
- Added a new CI job for running codespell.
- Fixed all detected typos.

### CODE REVIEW

**Comment by vr009** [comment]
I can extend this patch by adding a job to our CI and fixing all the typos in this repo.
Then I need @oleg-jukovec approve.

**Comment by oleg-jukovec** [comment]
> I can extend this patch by adding a job to our CI and fixing all the typos in this repo.

It will be great. I will be very grateful.

File: Makefile [hunk 1]
```diff
@@ -134,3 +134,9 @@ fuzzing:
 	@echo "Running fuzzing tests"
 	go clean -testcache
 	go test -tags "$(TAGS)" ./... -run=^Fuzz -v -p 1
+
+.PHONY: codespell
+codespell:
+	@echo "Running codespell"
+	pip3 install codespell
```
Comment (oleg-jukovec) [inline Makefile:141]: Could you make this line as a comment? We don't install any program in the Makefile (gofmt, golangci-lint). So let it be the same.
```suggestion
	# pip3 install codespell
```

**Review by oleg-jukovec — APPROVED** [review]
Thank you for the patch! LGTM, except 1 thing.

**Review by 0x501D — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (vr009) [reply Makefile:141]: May be it is worth to move the installation to deps target?

Comment (vr009) [reply Makefile]: Fixed.

File: .github/workflows/check.yaml [hunk 2]
```diff
@@ -48,3 +48,14 @@ jobs:
           # cause those comments get rendered in documentation by godoc.
           # See https://github.com/tarantool/go-tarantool/pull/160#discussion_r858608221
           args: -E goimports -D errcheck
+
+  codespell:
+    runs-on: ubuntu-latest
```
Comment (oleg-jukovec) [inline .github/workflows/check.yaml:53]: Oh, I've noticed a little problem: the job runs two times for a pull request. Could you make this change? See `luacheck` and `golangci-lint` jobs in the file.
```suggestion
    runs-on: ubuntu-latest
    if: |
      github.event_name == 'push' ||
      github.event_name == 'pull_request' &&
      github.event.pull_request.head.repo.full_name != github.repository
```

Comment (vr009) [reply .github/workflows/check.yaml:53]: Thank you! Fixed.

**Comment by oleg-jukovec** [comment]
Thank you for the patch!

**Pull request merged**